client/shipping-label/components/label-purchase-modal/index.js

```diff
@@ -9,13 +9,14 @@ import { translate as __ } from 'i18n-calypso';
  */
 import Modal from 'components/modal';
 import ActionButtons from 'components/action-buttons';
-import Spinner from 'components/spinner';
+import LoadingSpinner from 'components/loading-spinner';
 import getPDFSupport from 'lib/utils/pdf-support';
-import AddressStep from './steps/address';
-import PackagesStep from './steps/packages';
-import RatesStep from './steps/rates';
+import AddressStep from './address-step';
+import PackagesStep from './packages-step';
+import RatesStep from './rates-step';
 import Sidebar from './sidebar';
 import { getRatesTotal } from 'shipping-label/state/selectors/rates';
+import FormSectionHeading from 'components/forms/form-section-heading';
 
 const PrintLabelDialog = ( props ) => {
```

Comment:
Let's rename `PrintLabelDialog` to `PurchaseDialog` to match the others:

```
import PrintLabelDialog from './components/label-purchase-modal';
```

vs

```
import RefundDialog from './components/label-refund-modal';
import ReprintDialog from './components/label-reprint-modal';
```